docs/guides/setup/configuring-metadata-stores/how-to-configure-an-expectation-store-in-amazon-s3.md

```diff
@@ -1,5 +1,108 @@
+
+    :::warning
+    If you are also storing [Validations in S3](../configuring-metadata-stores/how-to-configure-a-validation-result-store-in-amazon-s3.md) or [DataDocs in S3](../configuring-data-docs/how-to-host-and-share-data-docs-on-amazon-s3.md),  please ensure that the ``prefix`` values are disjoint and one is not a substring of the other.
+    :::
+
+    ```yaml
+    expectations_store_name: expectations_S3_store
+
+    stores:
+        expectations_S3_store:
+            class_name: ExpectationsStore
+            store_backend:
+                class_name: TupleS3StoreBackend
+                bucket: '<your_s3_bucket_name>'
+                prefix: '<your_s3_bucket_folder_name>'
+    ```
+
+
+4. **Copy existing Expectation JSON files to the S3 bucket**. (This step is optional).
+
+    One way to copy Expectations into Amazon S3 is by using the ``aws s3 sync`` command.  As mentioned earlier, the ``base_directory`` is set to ``expectations/`` by default. In the example below, two Expectations, ``exp1`` and ``exp2`` are copied to Amazon S3.  Your output should looks something like this:
+
+    ```bash
+    aws s3 sync '<base_directory>' s3://'<your_s3_bucket_name>'/'<your_s3_bucket_folder_name>'
+
+    upload: ./exp1.json to s3://'<your_s3_bucket_name>'/'<your_s3_bucket_folder_name>'/exp1.json
+    upload: ./exp2.json to s3://'<your_s3_bucket_name>'/'<your_s3_bucket_folder_name>'/exp2.json
+    ```
+
+
+5. **Confirm that the new Expectations store has been added by running** ``great_expectations --v3-api store list``.
+
+    Notice the output contains two Expectation stores: the original ``expectations_store`` on the local filesystem and the ``expectations_S3_store`` we just configured.  This is ok, since Great Expectations will look for Expectations in the S3 bucket as long as we set the ``expectations_name`` variable to ``expectations_S3_store``.
+
+    ```bash
+    great_expectations --v3-api store list
+
+    - name: expectations_store
+    class_name: ExpectationsStore
+    store_backend:
+        class_name: TupleFilesystemStoreBackend
+        base_directory: expectations/
+
+    - name: expectations_S3_store
+    class_name: ExpectationsStore
+    store_backend:
+        class_name: TupleS3StoreBackend
+        bucket: '<your_s3_bucket_name>'
+        prefix: '<your_s3_bucket_folder_name>'
+    ```
+
+
+6. **Confirm that Expectations can be accessed from Amazon S3 by running** ``great_expectations --v3-api suite list``.
+
+    If you followed Step 4, The output should include the 2 Expectations we copied to Amazon S3: ``exp1`` and ``exp2``.  If you did not copy Expectations to the new Store, you will see a message saying no expectations were found.
+
+    ```bash
+    great_expectations --v3-api suite list
+
+    2 Expectation Suites found:
+     - exp1
+     - exp2
```

Comment:
Interesting...  In some PRs, you moved the yaml or bash directive all the way to the left, while leaving the content aligned as per the list item; in this one, these directives are lined up with the list items -- and they all work equally well (and seemingly the same rendering).

Reply:
Yup this one isn't as pretty but I was having some issues getting it to render well. Would you like me to update that or are we good for purposes of this PR?